### PR TITLE
Add valid ping types that are currently being reported as errors

### PR DESCRIPTION
**Pull request by trink**

### CODE REVIEW

**Comment by trink** [comment]
@whd r?

File: moz_telemetry/io_modules/decoders/moz_telemetry/ping.lua [hunk 1]
```diff
@@ -162,6 +162,10 @@ schemas["testpilottest"]                        = schemas.vacuous
 schemas["malware-addon-states"]                 = schemas.vacuous
 schemas["sync"]                                 = schemas.vacuous
 schemas["outofdate-notifications-system-addon"] = schemas.vacuous
+schemas["x-shield-studies"]                     = schemas.vacuous
+schemas["shield-study"]                         = schemas.vacuous
+schemas["x-contextual-feature-recommendation"]  = schemas.vacuous
+schemas["FirefoxOS"]                            = schemas.vacuous
```
Comment (mreid-moz) [inline moz_telemetry/io_modules/decoders/moz_telemetry/ping.lua]: Should we just default to using the vacuous schema for any type where we do not have a specific schema?

Comment (trink) [inline moz_telemetry/io_modules/decoders/moz_telemetry/ping.lua]: sigh... just detected another valid one "addon-install-blocked" I am a +1 for the vacuous catch all

Comment (whd) [inline moz_telemetry/io_modules/decoders/moz_telemetry/ping.lua]: To be clear, I was not a -1. I was simply of the opinion that this shouldn't be my decision.

**Comment by whd** [comment]
LGTM

**Pull request merged**